server.js

```diff
@@ -13,6 +15,19 @@ io.on('connection', (socket) => {
   })
 })
 
+app.post('/', (req, resp) => {
+  if (req.act === 'join') {
+    joinGame(resp);
+  }
+  if (req.act === 'word') {
+    startGame(req, resp)
+  }
+  if (req.act === 'guess') {
+    makeGuess(req, resp)
+  }
+  resp.status(400).json("Incorrect 'act' verb");
```

Comment:
Good lookin, although I don't think a user can hit this intentionally anyways (without changing the functionality with dev tools). Still good to have!

Reply:
True! I also included it for development purposes, too! Now we can see with more accuracy where things might go wrong.